.github/workflows/generate_markdown_files.yml

```diff
@@ -0,0 +1,16 @@
+name: Generate markdown files
+
+on: push 
+
+jobs: 
+  generate: 
+    runs-on: ubuntu-latest
+    steps:    
+      - name: "Checkout"
+        uses: actions/checkout@v4
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.2.4
+      - name: Install deps
+        run: pnpm i
+      - name: generate-docs
+        run: pnpm -r run api-extractor && pnpm -r run generate-docs && pnpm run fmt
```

Comment:
I think you can just call `pnpm run docs` since the docs command already exists in the root package.json